### DISCUSSION TITLE
Translations for trending regions

### DISCUSSION THREAD

**Discussion by GilgusMaximus** [FreeTubeApp/FreeTube#3621] [General]
**Topic**

- [x] Languages

**Discussion**
Hi everyone, so now that we implemented the option to select the region from which FreeTube gets it's trending videos, we also used the amazing project from [stefangabos](https://github.com/stefangabos/world_countries) to translate the country lists into the respective languages. This way you can select the country from a list in your language. However, the project does not feature every language that is available in FreeTube. If someone, who speaks a language that is not yet available, has the time to translate some of countries, we and the users of that language would appreciate it, if a translation could be started.

**How to translate the countries**
The countries were specifically not added to Weblate, because it would reset a lot of progress on existing languages and set some of the not yet activated languages further away from the 80% threshold. Therefore the translation of the country lists, have to be made in the same way as contributing code to the project.
1. Take a look at this feed and see whether someone is already working on your language. If not, please make a comment stating that you will work on the language. 
2. Fork the project
3. Clone your project fork locally
4. Copy the folder at the path `/static/geolocations/en-US/` 
5. Rename the folder according to the same abbreviation as seen in the locale folder `/static/locales/`
6. Start translating the name of the countries in the countries.json file
7. Create a Pull request when you are finished or are not going to work any further. If you do not translate all countries, please leave the English names in, so other people can translate these.

Thanks for the contribution in advance, we appreciate it a lot.

Currently supported are

    Arabic
    Basque
    Chinese (Simplified)
    Chinese (Traditional)
    Czech
    Danish
    Dutch
    English
    Estonian
    Finnish
    French
    German
    Greek
    Hungarian
    Italian
    Japanese
    Korean
    Lithuanian
    Norwegian
    Polish
    Portuguese
    Romanian
    Russian
    Slovak
    Spanish
    Swedish
    Thai
    Ukrainian

**Comment by rddim** [comment]
Start working on Bulgarian language translation
